helpdesk_mgmt/security/helpdesk_security.xml

```diff
@@ -22,7 +22,7 @@
         </record>
     </data>
     <data noupdate="1">
-        <record id="helpesk_ticket_personal_rule" model="ir.rule">
+        <record id="helpdesk_ticket_personal_rule" model="ir.rule">
```

Comment:
With this change you have now duplicated the same record rule. Please don't do this thing without migration scripts.